src/iotjs_binding.cpp

```diff
@@ -85,49 +71,39 @@ JObject::JObject(bool v) {
 
 JObject::JObject(int v) {
   _obj_val = JVal::Number(v);
-  _unref_at_close = false;
+  _unref_at_close = true;
```

Comment:
I think in the new API we need to unref even the simple variables. I know that is a no-op at the memont, but still it can be changed in the future. So releasing everything is better.